bootstrap/terraform/aws-bootstrap/aws-lb-controller.tf

```diff
@@ -162,7 +174,13 @@ resource "aws_iam_policy" "alb" {
                 "arn:aws:elasticloadbalancing:*:*:targetgroup/*/*",
                 "arn:aws:elasticloadbalancing:*:*:loadbalancer/net/*/*",
                 "arn:aws:elasticloadbalancing:*:*:loadbalancer/app/*/*"
-            ]
+            ],
+            "Condition": {
```

Comment:
I remember this causing the controller to break around product hunt, unless we're confident it's kosher, we should revert it

Reply:
That was with the tags not being able to be added. which now has a new policy entry below.